crates/topos/src/config/sequencer.rs

```diff
@@ -0,0 +1,90 @@
+use std::path::{Path, PathBuf};
+
+use figment::{
+    providers::{Format, Serialized, Toml},
+    Figment,
+};
+use serde::{Deserialize, Serialize};
+
+use crate::components::sequencer::commands::Run;
+use crate::config::Config;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct SequencerConfig {
+    /// SubnetId of the local subnet node, hex encoded 32 bytes starting with 0x
+    pub subnet_id: Option<String>,
+
+    // Subnet endpoint in the form [ip address]:[port]
+    // Topos sequencer expects both websocket and http protocol available
+    // on this subnet endpoint
+    #[serde(default = "default_subnet_jsonrpc_endpoint")]
+    pub subnet_jsonrpc_endpoint: String,
+
+    // Core contract address
+    #[serde(default = "default_subnet_contract_address")]
+    pub subnet_contract_address: String,
+
+    /// Base Uri of TCE node to call grpc service api
+    #[serde(default = "default_base_tce_api_url")]
+    pub base_tce_api_url: String,
+
+    /// Polygon subnet node data dir, containing `consensus/validator.key`, e.g. `../test-chain-1`
+    #[serde(default = "default_subnet_data_dir")]
```

Comment:
Default to this test directory now. We will have to change that and make it required once we add the actual `topos node up` logic to start `tce` and `sequencer`.